fastlane/lib/fastlane/actions/mailgun.rb

```diff
@@ -103,6 +108,7 @@ def self.mailgunit(options)
         RestClient.post "https://api:#{options[:apikey]}@api.mailgun.net/v3/#{sandbox_domain}/messages",
                         from: "#{options[:from]}<#{options[:postmaster]}>",
                         to: (options[:to]).to_s,
+                        "h:Reply-To": options[:reply_to],
```

Comment:
What happens when the value is nil? does the API handle this properly? Maybe this value shouldn't be passed when it's not set by the user.